### PR TITLE
Fix: docker container not removed when jest tests fail, or when scilla type check script fails

### PR DESCRIPTION
**Pull request by gb3h**
Small QOL improvement for those forking this repoing and running tests frequently.
Links to issue #122 

Simple fix makes the bash cleanup lines run even in the event of failure.

### CODE REVIEW

**Review by noel-yoo — APPROVED** [review]
LGTM

File: ci_scilla_type_check.sh [hunk 1]
```diff
@@ -6,19 +6,23 @@ checker="${scilla}bin/scilla-checker"
...
+    $check > /dev/null || { echo "type-checking failed: $file"; quit 1; }
+    $cleanup > /dev/null || { echo "cleanup failed: $file"; quit 1; }
     
     echo "PASS:$file"
 done
 
-rm_container=`docker stop $container | xargs docker rm`
-echo $rm_container
-
+quit 0
```
Comment (noel-yoo) [inline ci_scilla_type_check.sh:28]: @gb3h Can you add a new line?

**Review by noel-yoo — CHANGES_REQUESTED** [review]

File: ci_scilla_type_check.sh (hunk 1 above)
Comment (gb3h) [reply ci_scilla_type_check.sh]: Done 👍

**Review by noel-yoo — APPROVED** [review]
LGTM

**Review by bb111189 — APPROVED** [review]

**Pull request merged**